djaopsp/fixtures/content.json

```diff
@@ -510,6 +510,14 @@
   },
   "model": "survey.unit", "pk": 143
 },
+{
+  "fields": {
+    "slug": "chatgpt_answer",
+    "title": "ChatGPT Answer",
+    "system": 0
```

Comment:
Use `"system": 4` for `Unit.SYSTEM_FREETEXT` in order to store a text response in the database.